--- a/lib/python3.5/site-packages/msm/mycroft_skills_manager.py        2018-07-10 18:05:09.126435756 +0100
+++ b/lib/python3.5/site-packages/msm/mycroft_skills_manager.py        2018-07-10 19:00:12.275524635 +0100
@@ -23,6 +23,7 @@ import logging
 from glob import glob
 from itertools import chain
 from multiprocessing.pool import ThreadPool
+from os import environ
 from os.path import expanduser, join, dirname, isdir
 
 from typing import Dict, List
@@ -42,8 +43,13 @@ class MycroftSkillsManager(object):
     def __init__(self, platform='default', skills_dir=None, repo=None,
                  versioned=True):
         self.platform = platform
-        self.skills_dir = expanduser(skills_dir or '') \
-                          or self.DEFAULT_SKILLS_DIR
+
+        if environ.get('MYCROFT_SKILLS') is not None:
+            self.skills_dir = environ['MYCROFT_SKILLS']
+        else:
+            self.skills_dir = expanduser(skills_dir or '') \
+                              or self.DEFAULT_SKILLS_DIR
+
         self.repo = repo or SkillRepo()
         self.versioned = versioned
 
--- a/lib/python3.5/site-packages/msm/skill_entry.py
+++ b/lib/python3.5/site-packages/msm/skill_entry.py
@@ -227,9 +227,12 @@ class SkillEntry(object):
             tmp_location = mktemp()
             Repo.clone_from(self.url, tmp_location)
             self.is_local = True
-            Git(tmp_location).reset(self.sha or 'HEAD', hard=True)
         except GitCommandError as e:
             raise CloneException(e.stderr)
+        try:
+            Git(tmp_location).reset(self.sha or 'HEAD', hard=True)
+        except GitCommandError:
+            pass
 
         if isfile(join(tmp_location, '__init__.py')):
             move(join(tmp_location, '__init__.py'),
--- a/lib/python3.5/site-packages/msm/skill_repo.py  2018-07-10 19:26:50.774291041 +0100
+++ b/lib/python3.5/site-packages/msm/skill_repo.py  2018-07-10 19:26:14.933507937 +0100
@@ -20,7 +20,7 @@
 # specific language governing permissions and limitations
 # under the License.
 from glob import glob
-from os import makedirs
+from os import environ, makedirs
 from os.path import exists, join, isdir, dirname, basename
 
 from git import Repo
@@ -36,6 +36,9 @@ LOG = logging.getLogger(__name__)
 
 class SkillRepo(object):
     def __init__(self, path=None, url=None, branch=None):
+        if environ.get('MYCROFT_SKILLS_REPO') is not None:
+            path = environ['MYCROFT_SKILLS_REPO']
+
         self.path = path or "/opt/mycroft/.skills-repo"
         self.url = url or "https://github.com/MycroftAI/mycroft-skills"
         self.branch = branch or "18.08"
@@ -59,9 +62,13 @@ class SkillRepo(object):
 
         try:
             git.checkout(self.branch)
+        except GitCommandError as exc:
+            raise MsmException('Invalid branch: ' + self.branch)
+
+        try:
             git.reset('origin/' + self.branch, hard=True)
         except GitCommandError:
-            raise MsmException('Invalid branch: ' + self.branch)
+            pass
 
     def get_skill_data(self):
         """ generates tuples of name, path, url, sha """
